desktop/apps/artwork/components/auction_artworks/helpers.coffee

```diff
@@ -1,17 +1,29 @@
-_ = require 'underscore'
-masonry = require '../../../../components/artwork_masonry/index.coffee'
+masonry = require '../../../../components/artwork_masonry_4_column/index.coffee'
+moment = require 'moment'
+{ ARTWORK_DISPLAY_NUM } = require './config.coffee'
+{ include, shuffle, take } = require 'underscore'
 
-reduceArray = (artworks, rows) ->
-  elements = artworks.splice(0,3)
-  rows.push(elements)
+isLiveOpen = (status, live_start_at) ->
```

Comment:
If possible, we should use the `is_live_open` from [metaphysics](https://github.com/artsy/metaphysics/blob/0294f13484713ece7f3983ddda67f92297a6fd07/schema/sale/index.js#L88) here. I think elsewhere, we needed to compute it on-the-fly for a client-side clock, but as this is going to be a static component, it's better for us to grab this info directly from the API.

Also #minor but we call this field `auction_state` from the API, not `status` which is a little more descriptive.